libhb/nvenc_common.c

```diff
@@ -73,12 +127,44 @@ int hb_nvenc_h264_available()
 
 int hb_nvenc_h265_available()
 {
-    return hb_check_nvenc_available();
+    if (is_nvenc_hevc_available != -1)
+    {
+        return is_nvenc_hevc_available;
+    }
+    
+    if (!hb_check_nvenc_available()){
+        is_nvenc_hevc_available = 0;
+        return is_nvenc_hevc_available;
+    }
+    
+    if (hb_nvenc_get_cuda_version() >= 6.0) {
```

Comment:
Maxwell second generation seem to support HEVC to some degree: https://developer.nvidia.com/video-encode-and-decode-gpu-support-matrix-new. That would be compute capability 6.2>= https://en.wikipedia.org/wiki/CUDA

Reply:
Last I heard, maxwell cards don't work correctly with HB so we've just considered them unsupported. A driver update a while back seemed to have broken things.  I've not got an older 9 series card to check.

If they do infact work, I can bump this down to 5.2 and allow it through.